parallel-install/example/example.go

```diff
@@ -2,8 +2,9 @@ package main
 
 import (
 	"flag"
+	log "github.com/sirupsen/logrus"
```

Comment:
Do we need logrus here? Perhaps built-in log package is enough?

Reply:
I used it for testing purposes, I can remove it now :)